client/src/configs/env.ts

```diff
@@ -58,3 +58,6 @@ export const ENABLE_PRODUCT_CATALOGUE = process.env.REACT_APP_ENABLE_PRODUCT_CAT
 
 /** Activate the plans menu */
 export const ENABLE_PLANS = process.env.REACT_APP_ENABLE_PLANS === 'true';
+
+/** Activate the plans menu */
+export const ENABLE_LOCATIONS = process.env.REACT_APP_ENABLE_LOCATIONS === 'true';
```

Comment:
Has the env being added in env documentation?

Reply:
no its not where do i have to document this 🤔💭

Reply:
@ashfahan [docs/env.md](https://github.com/OpenSRP/web/blob/master/docs/env.md)

Reply:
done